site/content/blog/2022-11-30-wordpress-migration.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Wordpress to Flowershow migration tutorial
+description: Easily migrate your Wordpress content in 3 easy steps!
+created: 2022-11-30
+authors: [Philippe Du Preez]
+---
+
+Is your Wordpress site's speed comparable to a tortoise with cramping legs? 🐢 Server costs bleeding you dry? 🖥️ Lucky for you, migrating your content over to Flowershow has never been easier! Flowershow 🌷 is an **awesome** open source Next.js template that comes with batteries included 🔋
+
+## Here's how it works in 3 simple steps:
+
+> [!note]
+> You should have Node installed
+
+#### 1. Export your wordpress instance content to xml (**Export All**) using the in-built standard wordpress tool https://wordpress.com/support/export/
+
+#### 2. Download [this script](https://github.com/flowershow/wordpress-export-to-markdown-plus) and then run the following:
+
+- `cd wordpress-export-to-markdown-plus`
```

Comment:
Need to explain how i get the script: i think the best would be to do a release and explain that people should download the tarball or git clone the repo.

Then ... you cd and run the command (do you need to do npm install first?)